src/spead.py

```diff
@@ -213,6 +213,8 @@ def __init__(self, from_string=None, id=None, name='', description='', shape=[],
                     self.dtype_str = self._dtype_pack(ndarray)
                     self.shape = ndarray.shape if isinstance(ndarray, numpy.ndarray) else ndarray[1]
                     self.size = numpy.multiply.reduce(self.shape)
+                    if isinstance(ndarray, numpy.ndarray):
+                        self.fortran_order = ndarray.flags.f_contiguous and not ndarray.flags.c_contiguous
```

Comment:
There is a default (it is set to `False` on line 210). I don't want to start adding new features (particularly if we have no immediately requirement for them) to this pull request - it's meant to be a bug fix.